src/Tizen.AIAvatar/src/Extensions/AvatarExtension.cs

```diff
@@ -22,9 +22,9 @@
 
 namespace Tizen.AIAvatar
 {
-    internal static class AvatarExtension
+    public static class AvatarExtension
```

Comment:
Please add `[EditorBrowsable(EditorBrowsableState.Never)]` TAG before ACR.
You can see the list of API by the comments of bot.